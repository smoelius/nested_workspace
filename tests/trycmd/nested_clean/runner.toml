args = ["nested", "clean", "--config", "target.'cfg(all())'.runner = 'runner'"]

[bin]
name = "cargo-nested"

[fs]
cwd = "../../../fixtures/runner"
