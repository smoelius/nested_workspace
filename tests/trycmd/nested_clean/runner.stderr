     Removed [..] files[..]
<<< [CWD]/package
     Removed [..] files[..]
>>> [CWD]/package
